mysql_scripts/table_creation/FullDB.sql

```diff
@@ -80,7 +80,7 @@ CREATE TABLE AppointmentTime (
 	maximumNumberOfAppointments INTEGER UNSIGNED DEFAULT NULL,
 	percentageAppointments INTEGER UNSIGNED NOT NULL DEFAULT 100,
 	approximateLengthInMinutes INTEGER UNSIGNED NOT NULL DEFAULT 60,
-	CONSTRAINT percentageCheck CHECK (percentageAppointments>=0 AND percentageAppointments<=100),
+	CONSTRAINT percentageCheck CHECK (percentageAppointments>=0 AND percentageAppointments<=300),
```

Comment:
Hey funny story, so this constraint actually just doesn't exist in the PROD DB. Should we remove it from our table creation script? Or should I add it to the PROD DB? @CollinsSpencer

Reply:
Good point. I will add it to the DB, and update the front-end and back-end to match the 300% max.